clarifai/client/model.py

```diff
@@ -1296,20 +1297,52 @@ def _download_exported_model(
       model_export_url = get_model_export_response.export.url
       model_export_file_size = get_model_export_response.export.size
 
-      session = requests.Session()
-      retries = Retry(total=10, backoff_factor=1, status_forcelist=[500, 502, 503, 504])
-      session.mount('https://', HTTPAdapter(max_retries=retries))
-      session.headers.update({'Authorization': self.metadata[0][1]})
-      response = session.get(model_export_url, stream=True)
-      response.raise_for_status()
-
       with open(local_filepath, 'wb') as f:
         progress = tqdm(
             total=model_export_file_size, unit='B', unit_scale=True, desc="Exporting model")
-        for chunk in response.iter_content(chunk_size=8192):
-          f.write(chunk)
-          progress.update(len(chunk))
-        progress.close()
+        downloaded_size = 0
+        range_size = 31457280  # 30MB
```

Comment:
these variables like `range_size` and `chunk_size` are configurable so they can be read from a `constants` module

Reply:
Done